modules/installation-guide/partials/con_persistent-volume-configuration.adoc

```diff
@@ -5,95 +5,123 @@
+The workspace engine replaces volumes related to user-defined volumes with a subPath in the common volume. 
+SubPaths have a `_<workspace-ID>_` or `__<original-volume-name>__` prefix.
+See xref:how-subpaths-are-used-in-persistent-volumes_{context}[].
+The {prod-short} volume name is identical to the name of the user-defined PV.
+Therefore, if a workspace container uses a {prod-short} volume with the same name as the user-defined PV, they will use the same shared folder in the common PV.
 
-If there are several nodes, you can use the `common` strategy, but:
+When the user deletes a workspace, the workspace engine deletes the corresponding subdirectory (`$\{ws-id}`) in the PV directory.
 
-* The workspace PVC access mode must be reconfigured to `ReadWriteMany` (RWM), so multiple nodes can use this PVC simultaneously.
+When the user deletes the last workspace, the workspace engine removes the common PV. 
 
-* Only one workspace in the same {orch-namespace} may be running. See xref:installation-guide:configuring-the-number-of-workspaces-that-a-user-can-run.adoc[].
+Restrictions on the `common` storage strategy with the `ReadWriteOnce` access mode::
+The `ReadWriteOnce` access mode limits each user to run only one concurrent workspace.
+See xref:installation-guide:configuring-the-number-of-workspaces-that-a-user-can-run.adoc[].
 
-The `common` PVC strategy is not suitable for large multi-node clusters. Therefore, it is best to use it in single-node clusters. However, in combination with the `per-workspace` {orch-namespace} strategy, the `common` PVC strategy is usable for clusters with not more than 75 nodes. The PVC used with this strategy must be large enough to accommodate all projects to prevent a situation in which one project depletes the resources of others.
+Scalability::
+The `common` storage strategy is not suitable for multi-node clusters with the 'ReadWriteOnce' access mode for PVs if the number of concurrently running workspaces per user is more than 1.
 
-[id="the-per-workspace-pvc-strategy_{context}"]
-== The `per-workspace` PVC strategy
+Persistent volumes (PV) provisioning::
+Create a large enough PV to accommodate all projects to prevent a situation in which one project depletes the resources of others.
 
-The `per-workspace` strategy is similar to the `common` PVC strategy. The only difference is that all workspace Volumes, but not all the workspaces, use the same PVC as the default data storage for:
+[id="the-per-workspace-storage-strategy_{context}"]
+== The `per-workspace` storage strategy
 
-* projects
-* workspace logs
-* additional Volumes defined by a user
+Each workspace uses one dedicated PV. 
+All {prod-short} volumes defined within a single workspace use the same PV.
 
-With this strategy, {prod-short} keeps its workspace data in assigned PVs that are allocated by a single PVC.
+Scalability::
+Consider enabling this strategy on large multi-node clusters with a higher amount of users.
```

Comment:
I would probably drop this scalability recommendation at all. For me, it does not make much sense and it is much more ambiguous than useful. The scalability recommendation would be related to the number of PVs, which is already covered above